converters/fromZigbee.js

```diff
@@ -1767,6 +1767,18 @@ const converters = {
             }
         },
     },
+    QBKG25LM_click: {
+        cluster: 'genMultistateInput',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if ([1, 2].includes(msg.data.presentValue)) {
+                const mapping = {41: 'left', 42: 'center', 43: 'right'};
+                const times = {1: 'single', 2: 'double', 3: 'triple', 0: 'hold', 255: 'release'};
```

Comment:
Due to `if ([1, 2].includes(msg.data.presentValue)) {` only `1` and `2` make sense here, why are `3` `0` and `255` also here?

Reply:
I'm sorry, because I only used these two, so I missed, I added it

Reply:
> Due to `if ([1, 2].includes(msg.data.presentValue)) {` only `1` and `2` make sense here, why are `3` `0` and `255` also here?

I see you've already modified it.

Reply:
I didn't modify it, please update the code.